conf/zeppelin-site.xml.template

```diff
@@ -396,4 +396,18 @@
 </property>
 -->
 
+<!--
+<property>
+  <name>zeppelin.server.strict.transport</name>
+  <value>max-age=631138519</value>
+  <description>The HTTP Strict-Transport-Security response header is a security feature that lets a web site tell browsers that it should only be communicated with using HTTPS, instead of using HTTP. Enable this when Zeppelin is running on HTTPS.Set value is in Seconds equivalent to 20 years.</description>
+</property>
+-->
+<!--
+<property>
+  <name>zeppelin.server.xxss.protection</name>
+  <value>1; mode=block</value>
```

Comment:
I see - I understand the differences actually, but was wondering why the default value in the template was different from the default value in the code.

I feel either they should be the same, or they should be explained like you have in this PR. For the later approach, you could explain `1; mode=block` as recommended for stricter safety in the description, but that the default value in the absence of a value in `zeppelin-site.xml` is simply `1` which means ... etc.